examples/js/loaders/MMDLoader.js

```diff
@@ -2179,9 +2198,135 @@ THREE.MMDLoader.prototype.createMesh = function ( model, texturePath, onProgress
 			m.blendSrcAlpha = THREE.SrcAlphaFactor;
 			m.blendDstAlpha = THREE.DstAlphaFactor;
 
-			if ( p.envMap !== undefined ) {
+			if ( m.map !== null ) {
+
+				function checkTextureTransparency ( m ) {
+
+					m.map.readyCallbacks.push( function ( t ) {
+
+						// Is there any efficient ways?
+						function createImageData ( image ) {
+
+							var c = document.createElement( 'canvas' );
+							c.width = image.width;
+							c.height = image.height;
+
+							var ctx = c.getContext( '2d' );
+							ctx.drawImage( image, 0, 0 );
+
+							return ctx.getImageData( 0, 0, c.width, c.height );
+
+						};
+
+						function detectTextureTransparency ( image, uvs ) {
```

Comment:
MMD decides whether a certain material is transparent or not by
checking the alpha value actually used.
So I need to judge with uv.

For examples, assume
Material A and Material B uses the same Texture image partially including alpha < 1.0.
If Material A uses the part of the image including alpha < 1.0 while
Material B uses the part not including alpha < 1.0, 
then Material A will be transparent but Material B won't be.